src/main/java/net/dv8tion/jda/internal/entities/ReceivedMessage.java

```diff
@@ -129,10 +129,19 @@ public JDA getJDA()
         return api;
     }
 
+    @Nullable
     @Override
-    public Message getReferencedMessage()
+    public MessageReference getMessageReference()
     {
-        return referencedMessage;
+        return messageReference;
+    }
+
+    @Nullable
+    @Override
+    public Message getReferencedMessage() {
```

Comment:
Your brace is on the wrong line, also make this a default implementation in the `Message` interface

Reply:
Is this not a breaking change?

Reply:
Not really? Unless you use internals, then it could potentially break invokevirtual but usually not.